src/lsp/superbol_preprocs/main.ml

```diff
@@ -13,10 +13,10 @@
 
 module EXEC_MAP = Cobol_preproc.Options.EXEC_MAP
 
-let exec_scanners =
+let exec_scanners =  
```

Comment:
Extra spaces here. Please try to find whether there's a configuration setting in your editor that would automatically strip those extraneous characters.